### PR TITLE
Fix for the ocs-ci issue 7290

### PR DESCRIPTION
**Pull request by keemano**

### CODE REVIEW

File: tests/manage/pv_services/space_reclaim/test_rbd_reclaimspace_cronjob.py [hunk 1]
```diff
@@ -88,10 +89,11 @@ def test_rbd_space_reclaim_cronjob(self, pause_and_resume_cluster_load):
             )
             pod_obj.get_fio_results()
 
-        # Verify used size after IO
-        exp_used_size_after_io = used_size_before_io + (16 * self.pool_replica)
-        used_size_after_io = fetch_used_size(cbp_name, exp_used_size_after_io)
-        log.info(f"Used size after IO is {used_size_after_io}")
+            # Verify used size after IO
+            exp_used_size_after_io = temp_used_size + (16 * self.pool_replica)
```
Comment (jilju) [inline tests/manage/pv_services/space_reclaim/test_rbd_reclaimspace_cronjob.py:93]: Is this correct ?
```suggestion
            exp_used_size_after_io = temp_used_size + (4 * self.pool_replica)
```

File: tests/manage/pv_services/space_reclaim/test_rbd_reclaimspace_cronjob.py [hunk 2]
```diff
@@ -78,6 +78,7 @@ def test_rbd_space_reclaim_cronjob(self, pause_and_resume_cluster_load):
         log.info(f"Used size before IO is {used_size_before_io}")
 
         # Create four 4 GB file
+        temp_used_size = used_size_before_io
```
Comment (jilju) [inline tests/manage/pv_services/space_reclaim/test_rbd_reclaimspace_cronjob.py:81]: This is a duplicate variable. used_size_before_io is not used after this line.

File: tests/manage/pv_services/space_reclaim/test_rbd_reclaimspace_cronjob.py [hunk 3]
```diff
@@ -88,10 +88,11 @@ def test_rbd_space_reclaim_cronjob(self, pause_and_resume_cluster_load):
...
 
-        # Verify used size after IO
-        exp_used_size_after_io = used_size_before_io + (16 * self.pool_replica)
-        used_size_after_io = fetch_used_size(cbp_name, exp_used_size_after_io)
-        log.info(f"Used size after IO is {used_size_after_io}")
+            # Verify used size after IO
+            exp_used_size_after_io = used_size_before_io + (4 * self.pool_replica)
+            used_size_after_io = fetch_used_size(cbp_name, exp_used_size_after_io)
+            log.info(f"Used size after IO in {filename} is {used_size_after_io}")
+            used_size_before_io = used_size_after_io
```
Comment (keesturam) [inline tests/manage/pv_services/space_reclaim/test_rbd_reclaimspace_cronjob.py:95]: What is the purpose of this variable`used_size_before_io`?
Do we use it elsewhere?

File: tests/manage/pv_services/space_reclaim/test_rbd_reclaimspace_cronjob.py [hunk 4]
```diff
@@ -88,10 +88,11 @@ def test_rbd_space_reclaim_cronjob(self, pause_and_resume_cluster_load):
             )
             pod_obj.get_fio_results()
 
-        # Verify used size after IO
-        exp_used_size_after_io = used_size_before_io + (16 * self.pool_replica)
-        used_size_after_io = fetch_used_size(cbp_name, exp_used_size_after_io)
-        log.info(f"Used size after IO is {used_size_after_io}")
+            # Verify used size after IO
+            exp_used_size_after_io = used_size_before_io + (4 * self.pool_replica)
```
Comment (keesturam) [inline tests/manage/pv_services/space_reclaim/test_rbd_reclaimspace_cronjob.py:92]: (4 * self.pool_replica) 
why 4?

Comment (keemano) [reply tests/manage/pv_services/space_reclaim/test_rbd_reclaimspace_cronjob.py:92]: > (4 * self.pool_replica) why 4?

4GB is the file size. Previously we were calculating total size after completing ios on all 4 files (4*4 = 16Gb), now I changed to calculate the size after completing ios immediately for all the files in each iteration, so it has to 4GB not 16GB

File: tests/manage/pv_services/space_reclaim/test_rbd_reclaimspace_cronjob.py (hunk 3 above)
Comment (keemano) [reply tests/manage/pv_services/space_reclaim/test_rbd_reclaimspace_cronjob.py:95]: > What is the purpose of this variable`used_size_before_io`? Do we use it elsewhere?
"used_size_before_io" is the size of the pool before the actual ios that test executes.

Comment (keemano) [reply tests/manage/pv_services/space_reclaim/test_rbd_reclaimspace_cronjob.py:95]: > Do we use it elsewhere?

Yes, we use this variable every time to calculate the pool size after running ios on each file. Please look at line no 92.

**Pull request closed without merging**